uisdk/src/adyen/java/com/karhoo/uisdk/AdyenPaymentPresenter.kt

```diff
@@ -104,7 +105,7 @@ class AdyenPaymentPresenter(
                 is DropInResult.Error -> { // Is handled below
                 }
                 is DropInResult.CancelledByUser -> view?.refresh()
-                is DropInResult.Finished -> { // No need to handle this case, it will not occur if resultIntent is specified
+                is DropInResult.Finished -> { analytics?.cardAuthorisationSuccess(quote?.id)// No need to handle this case, it will not occur if resultIntent is specified
```

Comment:
if this case will not occur, is this the best place to trigger the auth success event?